TLM/TLM/State/OptionsTabs/OptionsGeneralTab.cs

```diff
@@ -243,7 +243,9 @@ public static class OptionsGeneralTab {
                 = string.Format(
                     T("General.Tooltip.Format:Window transparency: {0}%"),
                     GlobalConfig.Instance.Main.GuiOpacity);
-            _guiOpacitySlider.RefreshTooltip();
+            if (LoadingExtension.IsGameLoaded) {
```

Comment:
How does this solve issue of the update happening in _wrong thread_?

Reply:
While loading the savegame, slider values are set via this setter, the condition prevents the GUI update from executing which solves the hard crash. A more elegant way would be to move GUI update out of these functions.